spec/libraries/azure_engine_spec.rb

```diff
@@ -28,12 +31,14 @@ def delete_message(queue_name, message_object_id, message_object_pop_receipt)
   module Entity
     module Queue
       class DummyMessage
-        def id
-          1
-        end
-
-        def pop_receipt
-          ''
+        attr_accessor :id
+        attr_accessor :message_text
+        attr_accessor :pop_receipt
+
+        def initialize
```

Comment:
__[method_definition_validator]__
* Lint/DefinitionValidator: pop_receipt is undefined. Did you mean? initialize at [lib/slide_hub/cloud/engine/azure.rb#L60](/ryuzee/SlideHub/blob/780b037b1ace65f30d670708afd450dd42a6af75/lib/slide_hub/cloud/engine/azure.rb#L60)

<!-- Comment by [SideCI](https://sideci.com) -->